### ISSUE TITLE
Epochs of running

### ISSUE THREAD

**Issue by tristan199714** [furkanbiten/GoodNews#14]
How many epochs should I run your code with to get the result published in your paper?

**Comment by KhairulM** [comment]
wanted to know this too, I've run it for 34 epoch now, but the training loss is still around 4.x
